unity/EditorPlugin/PluginSettings.cs

```diff
@@ -262,7 +261,7 @@ private static void RiderPreferencesItem()
       {
         //UnityEditorInternal.InternalEditorUtility.OpenFileAtLineExternal(PluginEntryPoint.LogPath, 0);
         // works much faster than the commented code, when Rider is already started
-        PluginEntryPoint.OpenAssetHandler.OnOpenedAsset(PluginEntryPoint.LogPath, 0, 0);
```

Comment:
this can't be changed unfortunately
used from the package https://github.cds.internal.unity3d.com/unity/com.unity.ide.rider/blob/80e35461c5bed207859e3bda4d1006d9cb79afbc/Packages/com.unity.ide.rider/Rider/Editor/EditorPluginInterop.cs#L71

Reply:
Done